galsim/config/gsobject_ring.py

```diff
@@ -44,7 +45,7 @@ def _BuildRing(config, base, ignore, gsparams, logger):
         full_rotation = math.pi * galsim.radians
 
     dtheta = full_rotation / num
-    if logger and logger.isEnabledFor(logging.DEBUG):
+    if False:
```

Comment:
Thanks.  This (and a few others like it) happened when I was trying to figure out why the logging commands were so slow at times.  I put them all back in now as part of this commit: f62f5f47e1102a2180bfe902b4755c9b86371301